### ISSUE TITLE
Please close all issues due to audio overlap escalation.

### ISSUE THREAD

**Issue by WesleyTRV** [TheRealJoelmatic/RemoveAdblockThing#546]
This is a bit too much, just please address it.

**Comment by GroggyOtter** [comment]
Yup.  
And the temp fix guide I posted is pushed back like 3 pages b/c people are just blindly submitting the same reports over and over instead of commenting/participating in the posts that are already up.  
Imagine pushing a temporary solution back to a place where others can no longer find it.  
Good intentions with negative consequences.

Link to temporary solution:  
[❗Temp Fix To Current v5.4/v5.5 problem❗ - Downgrade to v5.3 [Text and video instructions]](https://github.com/TheRealJoelmatic/RemoveAdblockThing/issues/460) 

Doing this makes YouTube useable until 5.6 comes out and (hopefully) fixes everything.    

Some people are saying "Downgrade to 3.8, instead!"  
I've been downgraded to 5.3 since 5.4 was released and it has worked flawlessly for me.  
Win10 using the current version of Chrome.

Problems some people have mentioned:  
* "Didn't work for me":  
Did you verify the script is downgraded to 5.3?  
Did it get updated back to 5.4 or 5.5? Clicking the update button or having auto-update turned on?  
* "Double audio is still playing.":  
You're definitely still on 5.4 or 5.5 b/c that's when that specific bug started happening. Downgrade to 5.3.
* "Short ads are playing at the beginning":  
Some people have reported this. I, too, have had this happen with other versions of RAT. It's not specific to this version. I don't think they know what causes this b/c it's been around for a while now.  
On a good note, it **does** stop happening.  
Refreshing the page helps (I haven't had this bug happen to me for months).
